src/lib/components/SideBar/SideBar.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  import { getDrawerStore } from "@skeletonlabs/skeleton";
+
+  import SideBarFooter from "./SideBarFooter.svelte";
+  import SideBarLink from "./SideBarLink.svelte";
+
+  import CalendarIcon from "~icons/material-symbols/calendar-clock";
+  import CloseIcon from "~icons/material-symbols/close-rounded";
+  import SettingsIcon from "~icons/material-symbols/settings-rounded";
+
+  export let displayCloseButton = false;
+
+  const drawerStore = getDrawerStore();
+
+  const closeDrawer = drawerStore.close;
+</script>
+
+<div
+  id="sidebar"
+  class="flex flex-col bg-secondary-400 outline-secondary-800 rounded-l-lg rounded-r-none card outline-5 h-full w-full"
```

Comment:
```suggestion
  class="flex flex-col w-full h-full rounded-r-none rounded-left-lg md:rounded-none bg-secondary-400 outline-secondary-800 card outline-5"
```
Left roundness was being applied on larger screens